packages/shared/src/styles/base.css

```diff
@@ -661,6 +670,14 @@ html.light .invert .invert,
   position: relative;
   flex-direction: column;
   align-items: stretch;
+
+  @screen mobileL {
+    flex: 1;
+  }
+
+  @screen tablet {
+    flex: unset;
```

Comment:
This as well

Reply:
This just nulls out the one set above so not much concerns on it